junction/templates/pages/home_page.html

```diff
@@ -7,6 +7,11 @@
 
 {% block page_classes %}{{ block.super }} homepage{% endblock %}
 {% block breadcrumbs %}{% endblock %}
+
+{% block extra_css %}
+<link rel="stylesheet" href="{% static 'css/homepage.css' %}">
```

Comment:
This isn't used anywhere. Can be removed.

Reply:
ok

Reply:
removed please check